apps/angular/1-projection/src/app/component/teacher-card/teacher-card.component.ts

```diff
@@ -1,25 +1,47 @@
-import { Component, inject, OnInit } from '@angular/core';
-import { FakeHttpService } from '../../data-access/fake-http.service';
+import { NgOptimizedImage } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  inject,
+  OnInit,
+} from '@angular/core';
+import {
+  FakeHttpService,
+  randTeacher,
+} from '../../data-access/fake-http.service';
 import { TeacherStore } from '../../data-access/teacher.store';
 import { CardType } from '../../model/card.model';
 import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-teacher-card',
   template: `
     <app-card
       [list]="teachers()"
-      [type]="cardType"
-      customClass="bg-light-red"></app-card>
+      [itemTemplate]="itemTemplate"
+      class="bg-light-red"
+      (onAddNewItem)="addNewTeacher()">
+      <img priority ngSrc="assets/img/teacher.png" width="200" height="200" />
+    </app-card>
+
+    <ng-template #itemTemplate let-item>
+      <app-list-item
+        [name]="item.firstName"
+        [id]="item.id"
+        [type]="cardType"
+        (onDeleteItem)="deleteTeacher($event)"></app-list-item>
```

Comment:
very good
You don't need to pass id, when you get the onDeleteItem, you can pass item.id to the function in the parent